lib/src/r_tree/r_tree.dart

```diff
@@ -61,10 +62,277 @@ class RTree<E> {
     return _root.search(searchRect, shouldInclude);
   }
 
-  _growTree(Node<E> node1, Node<E> node2) {
-    NonLeafNode<E> newRoot = NonLeafNode<E>(_branchFactor);
-    newRoot.addChild(node1);
-    newRoot.addChild(node2);
+  /// Bulk adds all [items] to the rtree. This implementation draws heavily from
+  /// https://github.com/mourner/rbush and https://github.com/Zverik/dart_rbush.
+  load(List<RTreeDatum<E>> items) {
+    if (items.isEmpty) {
+      return this;
```

Comment:
Why is `this` returned here, but not elsewhere (lines 76 & 100)?

Reply:
because why doesn't this have a return type too